src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php

```diff
@@ -366,7 +366,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
     private function buildAttributeTypes(FormBuilderInterface $container): FormBuilderInterface
     {
-        return $this->attributeTypeFactory->build($container);
+        return $this->attributeTypeFactory->build($container, Constants::TYPE_OPENID_CONNECT_TNG);
```

Comment:
Not for now, but maybe an idea for the future to divide a large builder into smaller methods to apply structure,  methods like buildMetadata, buildOrganization, buildAttributesTypes, etc